edward/util/random_variables.py

```diff
@@ -699,3 +699,20 @@ def get_variables(x, collection=None):
     nodes.update(node.op.inputs)
 
   return list(output)
+
+
+def generate_multinomial_prediction(x, total_count=1):
+    """Generate a single multinomial prediction from a random variable,
+    comprised of `total_count` draws.
+
+    Args:
+      x: RandomVariable
+      total_count: number of draws to sample from `x`; the `n` in a Multinomial distribution.
```

Comment:
Is `total_count` the parameter in `x`, or is it the number of draws to sample from `x`? We refer to the latter by `sample_shape`. For example:
```python
Multinomial(total_count=2.0, probs=[0.25, 0.25, 0.5], sample_shape=5)
## <ed.RandomVariable 'Multinomial/' shape=(5, 3) dtype=float32>
```
represents 5 draws from a multinomial distribution.

Reply:
`total_count` is the parameter in `x`. I'll make this more clear.